conf/layer.conf

```diff
@@ -0,0 +1,10 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH =. "${LAYERDIR}:"
+
+# We have recipes-* directories, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+            ${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "meta-xt-products"
```

Comment:
this is by intention: every product will have the same collection named "meta-xt-products"
this way it is easier to track what belongs to the build system or specific product